src/fec/src/fec_conv.c

```diff
@@ -100,7 +100,7 @@ int fec_conv_encode(fec _q,
 
             // compute parity bits for each polynomial
             for (r=0; r<_q->R; r++) {
-                byte_out = (byte_out<<1) | parity(sr & _q->poly[r]);
+                byte_out = (byte_out<<1) | ((_q->poly[r] < 0) ^ parity(sr & abs(_q->poly[r])));
```

Comment:
real minor thing, but wouldn't this change need at least one additional unit-test?

Reply:
I kinda thought of that, but since polynomials are hardcoded and I did not want to change the defaults, I left it as it.